src/mds/MDCache.cc

```diff
@@ -11997,6 +12001,8 @@ void MDCache::repair_dirfrag_stats_work(MDRequestRef& mdr)
   if (!good_fragstat) {
     if (pf->fragstat.mtime > frag_info.mtime)
       frag_info.mtime = pf->fragstat.mtime;
+    if (pf->fragstat.change_attr > frag_info.change_attr)
+      frag_info.change_attr = pf->fragstat.change_attr;
```

Comment:
My understanding (quite possibly wrong) is that a client can see the frag attrs whenever it has the shared or exclusive cap for it. To actually _view_ the change_attr though, you would need to do a gather operation and the change_attrs would get synched up at that point.

So I don't think this is problematic, but please do sanity check me here.